src/App.tsx

```diff
@@ -1,16 +1,25 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
+
+import { Movie } from './types/Movie';
+
```

Comment:
I suggest to remove this spaces because don't see huge sence in them:
```suggestion
import { Movie } from './types/Movie';
```